control-plane/agents/core/src/core/wrapper.rs

```diff
@@ -147,14 +147,22 @@ impl NodeWrapper {
     }
 
     /// Reload the node by fetching information from mayastor
-    pub(crate) async fn reload(&mut self) -> Result<(), SvcError> {
+    pub(crate) async fn reload(&mut self, registry: &Registry) -> Result<(), SvcError> {
```

Comment:
what does it mean? "reload a node" ? it seems that we are re-reading its configuration, refresh? reoad_config() ?

Reply:
I decided to leave the name `reload` for now. I'll look at changing this in the next set of changes. It's effectively getting the latest runtime configuration from Mayastor.